src/ReviewPage.js

```diff
@@ -1,17 +1,81 @@
 import React, { useState } from "react";
 import "./CSS/reviewpage.css";
-import { MDBInput } from "mdb-react-ui-kit";
+import { MDBInput, MDBScrollspyNavLink } from "mdb-react-ui-kit";
 import axios from "axios";
+import UserPage from "./UserPage";
+import AlbumTable from "./Tables/AlbumTable";
+import {
+  useNavigate,
+  Navigate,
+} from "react-router-dom";
 
 function ReviewPage(props){
+
+  const [user, setUser] = useState({ albums: []
+
+  });
+
+  const [album, setAlbum] = useState({ album_name: ""
+
+  });
+
+  const [reviewData, setReview] = useState({ 
+    owner: props.userData.username,  
+    content: "" , 
+    rating: "",
+    reviewedItem: ""});
+
+  let navigate = useNavigate();
+
   var URL = 'http://localhost:5000';
   // var URL = 'https://musiccatalogbe.herokuapp.com';
 
   if (process.env.NODE_ENV === "production"){
     URL = 'https://musiccatalogbe.herokuapp.com';
   }
 
+
+  function handleChange(event) {
+    const { name, value } = event.target;
+    if (name === "album_name") setAlbum({ ...album, album_name: value });
+    //else if (name === "album") setReview({ ...reviewData, album: value });
+    else if (name === "review") setReview({ ...reviewData, review: value });
+    else if (name === "rating") setReview({ ...reviewData, rating: value });
+  }
+
+  function removeAlbum(index) {
+    const updated = user.albums.filter((character, i) => {
+      return i !== index;
+    });
+    setUser({ ...user, albums: updated });
+  }
+
+  async function getAlbum(album_name) {
+    try {
+      const response = await axios.get(
+        `${URL}/search/album/${album_name}`
+      );
+      return response.data;
+    } catch (error) {
+      console.log(error);
+      return false;
+    }
+  }
+
+  async function submitAlbum() {
+    const name = album.album_name;
+    const album_response = await getAlbum(name);
+    console.log(name)
+    const album_data = album_response.albums.items[0];
+    if (album_data !== undefined) {
```

Comment:
another opportunity for undefined errors, but this time with components as well. Once our functionality is done we should revisit this

Reply:
we can revisit once we figure out more error catching issues